commec/config/screen_tools.py

```diff
@@ -4,12 +4,13 @@
 Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
 """
 import os
+import logging
 from commec.config.io_parameters import ScreenIOParameters
-from commec.tools.blastn_handler import BlastNDataBase
-from commec.tools.blastx_handler import BlastXDataBase
-from commec.tools.blastdmnd_handler import DiamondDataBase
-from commec.tools.cmscan_handler import CmscanDataBase
-from commec.tools.hmm_handler import HMMDataBase
+from commec.tools.blastn_handler import BlastNHandler
+from commec.tools.blastx_handler import BlastXHandler
+from commec.tools.blastdmnd_handler import DiamondHandler
+from commec.tools.cmscan_handler import CmscanHandler
+from commec.tools.hmm_handler import HMMHandler
 
 class CommecDatabases():
```

Comment:
Then this becomes `ScreenTools`

and we get `self.biorisk_screen_tool`, `self.protein_screen_tool` (or `self.biorisk_tool`), yeah?